tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": ["dom", "dom.iterable", "esnext", "es6"],
```

Comment:
why do we need this option? is it because of the `find` method usage? 🤔

Reply:
I thought that initially but now is working fine without it. It was all about I had to use an array